README.md

```diff
@@ -91,6 +91,22 @@ NO:
 event UpdatedOwner(address newOwner);
 ```
 
+##### C. Order event parameters in chronological order.
```

Comment:
I think "chronological" could be confusing 🤔 . Might make more sense to say like, "Events logging updated values should follow the convention: `previous, new`

Reply:
yeah I agree I think it's confusing, I've just updated it - thanks